src/binary_converter.py

```diff
@@ -0,0 +1,13 @@
+def dec_bin(num: int) -> int:
+    binary = ''
+    while num>0:
+        if num%2 == 0:
+            binary = '0' + binary
+        else:
+            binary = '1' + binary
+        num = int(num/2)
+    return binary
+
+number = int(input("Insert first number: "))
+
+print("The binary number is: "+dec_bin(number))
```

Comment:
metti queste righe dentro `if __name__ == "__main__"` https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do